app/tests/system/nightwatch-config.js

```diff
@@ -21,35 +22,40 @@ module.exports = {
     'test_settings': {
         'default': {
             'globals' : {
-                'waitForConditionTimeout' : 60000
+                'waitForConditionTimeout' : 60000,
+                'waitForConditionPollInterval': 500
             },
+            'end_session_on_fail': false,
             'launch_url': 'http://localhost:4723/wd/hub',
             'selenium_host': 'localhost',
             'selenium_port': 4723,
             'silent': true,
             'output': true,
             'screenshots': {
-                'enabled': false,
-                'path': ''
+                'enabled': true,
+                'path': '${CIRCLE_ARTIFACTS}',
```

Comment:
Can we make this usable for local test runs also?  Maybe our `ios-appium.sh` script can set up these variables (or better yet, different variables that use the $CIRCLE_* ones as a source if they're defined)?